src/components/PlioQuestion.vue

```diff
@@ -77,7 +78,7 @@
 
           <!-- close button -->
           <button
-            v-else
+            v-if="!showButtonLoading && isAnswerSubmitted"
```

Comment:
`close` button (which comes after pressing `submit` button) will show only if the answer has been submitted and loading button is not showing